app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, file_in, file_out_with_rout = command.split()
+    if command == "mv":
+        rout_list = file_out_with_rout.split("/")
+        file_out_with_rout_cp = os.path.join(*rout_list)
```

Comment:
Do you really need this?
Will it work without these two lines of code?

Reply:
![image](https://user-images.githubusercontent.com/46852739/220172376-3e9fe520-62ac-45be-a406-4f8fa4c04035.png)
this is from checklist for this task

Reply:
You need this if you don't use os.renames(). For example, you separately create directories and rewrite the file. But in this case your code will work with string like this
`move_file("mv file.txt first_dir/second_dir/third_dir/file2.txt")`
also with string like this
move_file("mv file.txt first_dir\\second_dir\\third_dir\\file2.txt")